.travis.yml

```diff
@@ -9,12 +9,11 @@ matrix:
 
 env:
   global:
-  - COMPOSER_BIN_DIR=$(composer global config bin-dir --absolute)
+    - COMPOSER_BIN_DIR=$(composer global config bin-dir --absolute)
```

Comment:
I personally don't mind having the `-` at the same level, but in this file we had both so I opted for one, can be reverted to adopt the other way instead